tests/simple_test.rs

```diff
@@ -1,7 +1,9 @@
-use ft_io::*;
+mod utils;
+
 use gstd::{ActorId, BTreeMap, Encode};
 use gtest::{Program, System};
 use staking_io::*;
+use utils::token::*;
```

Comment:
Better to import more specific items.
```suggestion
use utils::token::FToken;
```